src/uPlot.js

```diff
@@ -1343,8 +1343,8 @@ export default function uPlot(opts, data, then) {
 	const drag = FEAT_CURSOR && cursor.drag;
 
 	let dragX = FEAT_CURSOR && drag.x;
-	let dragY = FEAT_CURSOR && drag.y;;
-
+	let dragY = FEAT_CURSOR && drag.y;
+	
```

Comment:
please enable trailing whitespace trimming in your editor.

Reply:
:+1: